### ISSUE TITLE
Feature: Reset Position

### ISSUE THREAD

**Issue by PtrckM** [keycastr/keycastr#267]
There are times I cant find where is the current position when using multiple monitors. It is a good feature to have a settings to reset the position to the default.

**Comment by akitchen** [comment]
Yes, it is possible that the visualizer can end up offscreen if there are changes to the resolution or number of displays attached. There are some internal heuristics to help recover but they are not always good enough. Contributions are welcome, whether a reset button or improvements to the offscreen detection.

**Comment by akitchen** [comment]
If you've really lost your visualizer you can reset all of KeyCastr's settings with the following command in the terminal:

`defaults delete io.github.keycastr`
